### PR TITLE
Adds basic website

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This creates a basic website which we can add logos etc to later! I will handle netlify and dns for func-e.io post-merge.

Screen Shot 2021-07-06 at 2 13 01 PM

### CODE REVIEW

File: .bingo/Variables.mk [hunk 1]
```diff
@@ -35,6 +35,12 @@ $(GORELEASER): $(BINGO_DIR)/goreleaser.mod
 	@echo "(re)installing $(GOBIN)/goreleaser-v0.166.0"
 	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=goreleaser.mod -o=$(GOBIN)/goreleaser-v0.166.0 "github.com/goreleaser/goreleaser"
 
+HUGO := $(GOBIN)/hugo-v0.85.0
```
Comment (mathetake) [inline .bingo/Variables.mk:38]: nice 😄

File: internal/cmd/usage_md_test.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/tetratelabs/func-e/internal/globals"
 )
 
-const siteMarkdownFile = "../../site/usage.md"
+const siteMarkdownFile = "../../USAGE.md"
```
Comment (mathetake) [inline internal/cmd/usage_md_test.go:28]: is this going to break the existing redirect from getenvoy.io?

Comment (mathetake) [reply internal/cmd/usage_md_test.go:28]: though it's not a problem at all😄

**Review by mathetake — APPROVED** [review]

File: internal/cmd/usage_md_test.go (hunk 2 above)
Comment (codefromthecrypt) [reply internal/cmd/usage_md_test.go:28]: yep it will and I'll fix it before anyone notices :D

**Comment by codefromthecrypt** [comment]
thanks for the fast turnaround! we need this in order to setup a stable homebrew. I'll follow-up the loose ends on site details before that now.

File: internal/cmd/usage_md_test.go (hunk 2 above)
Comment (codefromthecrypt) [reply internal/cmd/usage_md_test.go:28]: https://github.com/tetratelabs/getenvoy.io/pull/182

**Pull request merged**